### PR TITLE
JFokus post.

### PR DESCRIPTION
**Pull request by jdijt**

### CODE REVIEW

**Review by wjglerum — DISMISSED** [review]
nice one, thanks!

**Review by PeuTit — DISMISSED** [review]

File: posts/2024-03-15-jfokus-2024.adoc [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+== The Conference
+JFokus is held at the Waterfront Conference Center in Stockholm, located almost on top of the Stockholm Central Station.
+Catering wise the quality is excellent and the supply of coffee appears to be infinite which is always a good thing.
+Overall attending the conference was a very smooth and comfortable experience.
+I would like to compliment the organizers and volunteers on the quality of the event!
+
+Content wise a broad range of subjects is covered with a focus on java, the JVM, and the surrounding ecosystem.
+This year there were also a good number of talks discussing AI, both how to use it within your projects and how to use to it in your own coding process.
+With over 70 talks scheduled attending all the talks that look interesting is an impossible assignment!
```
Comment (lukewoodcock) [inline posts/2024-03-15-jfokus-2024.adoc:23]:
```suggestion
With over 70 talks scheduled, attending all the talks that look interesting is an impossible assignment!

File: posts/2024-03-15-jfokus-2024.adoc [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+Luckily all the talks are recorded and published on YouTube.
```
Comment (lukewoodcock) [inline posts/2024-03-15-jfokus-2024.adoc:24]:
```suggestion
Luckily all the talks were recorded and are published on YouTube.
```

Perhaps reference the link to the youtube videos you've provided at the end of the blog?

File: posts/2024-03-15-jfokus-2024.adoc [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+
+Content wise a broad range of subjects is covered with a focus on java, the JVM, and the surrounding ecosystem.
+This year there were also a good number of talks discussing AI, both how to use it within your projects and how to use to it in your own coding process.
+With over 70 talks scheduled attending all the talks that look interesting is an impossible assignment!
+Luckily all the talks are recorded and published on YouTube. 
+The first recordings were already online within a few days after the conference in some cases.
+
+== Talk: Tame Microservices with Contract Testing by Holly Cummins
+
+Personally I had never worked with contract tests before and I was curious how they could work.
```
Comment (lukewoodcock) [inline posts/2024-03-15-jfokus-2024.adoc:29]:
```suggestion
Personally I have never worked with contract tests before and I was curious how they work.

**Review by lukewoodcock — DISMISSED** [review]

**Review by wjglerum — APPROVED** [review]

**Pull request merged**